sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubProducerClientTest.java

```diff
@@ -179,45 +177,6 @@ public void sendStartSpanSingleMessage() {
         verify(tracer1, times(2)).end(eq("success"), isNull(), any());
     }
 
-    /**
```

Comment:
Same. This test can be kept. just ensure no links are added so that code path is tested.